### PR TITLE
add docs on debugging job event perf

### PR DESCRIPTION
**Pull request by chrismeyersfsu**
related to https://github.com/ansible/tower/issues/4985

### CODE REVIEW

**Comment by AlanCoding** [comment]
So conflict with minor change https://github.com/ansible/awx/pull/10348 is because you moved the file. Rebase might do the right thing.

File: docs/debugging/debugging_job_event_performance.md [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
...ir life in Ansible. Specifically, in the awx Ansible callback plugin. AWX creates JSON payloads out of function calls, and the parameters passed, to the registered callback plugin. The moment the JSON payload is created in the callback plugin is the `created` time on a job event record that is stored in the AWX database. The `modified` time on the job event record is the time the job event record was stored in the database. The difference in `modified - created` is the time it take for an event to go from Ansible and into the AWX database. The pieces in between that path are: (1) Ansible (2) the AWX callback plugin (inside of ansible-runner) (3) push to redis. This ends the processing that happens in the Ansible playbook process space. The work picks back up in the callback receiver process with a (4) pop from redis, (5) bulk insert to postgres, (6) emit event over websocket via redis. Again, the handling of the event changes process space here to (7) deliver over the websocket(s) via daphne to the other AWX nodes & to websocket browser clients. The (8) UI then gets the websocket event and displays it in the job details view. This is what we call the "critical path" for job events.
```
Comment (AlanCoding) [inline docs/debugging/debugging_job_event_performance.md]: > AWX creates JSON payloads out of function calls, and the parameters passed, to the registered callback plugin.

"to the registered callback plugin" --> "_in_ the registered callback plugin"?

side note: some of the parameters passed are python objects, so JSON-ifies some opinionated stringified version of them.

Comment (AlanCoding) [inline docs/debugging/debugging_job_event_performance.md]: > he difference in `modified - created` is the time it take for an event to go from Ansible and into the AWX database.

Tense, "took" vs "take" vs "takes"

File: docs/debugging/debugging_job_event_performance.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+  6. Events are emitted over websocket via Redis
+  7. At this point, the handling of the event changes process space to deliver over the websocket(s) via `daphne` to the other AWX nodes & to websocket browser clients
+  8. The UI then gets the websocket event and displays it in the job details view
+
+This is what we call the "critical path" for job events.
+
+### Quick Debugging
+
+* Log in to an AWX instance and open Chrome Debug Tools
+* Paste the above in
```
Comment (beeankha) [inline docs/debugging/debugging_job_event_performance.md]: What is the thing that you want people to paste in?

**Review by beeankha — APPROVED** [review]

**Pull request merged**